learn/dapps-list/dapp-directory.md

```diff
@@ -0,0 +1,309 @@
+
+#### URLs {: #urls }
+
+The `urls` property name/value pairs are used so a project can provide links to their website, socials, etc.
+
+The following table lists the supported `urls` properties:
+
+| Property Name |                                                    Description                                                     |                     Example                     |
+|:-------------:|:------------------------------------------------------------------------------------------------------------------:|:-----------------------------------------------:|
+|   `website`   |                                          The main website for the project                                          |            https://moonbeam.network/            |
+|     `try`     | URL a user should visit if they want to try out the dApp. Typically, this page will have a link to launch the dApp |            https://moonbeam.network/            |
+|   `twitter`   |                                           The project's Twitter profile                                            |       https://twitter.com/MoonbeamNetwork       |
+|   `medium`    |                                             The project's Medium site                                              |       https://medium.com/moonbeam-network       |
+|  `telegram`   |                                               The project's Telegram                                               |         https://t.me/Moonbeam_Official          |
+|   `github`    |                                          The project's GitHub repository                                           | https://github.com/moonbeam-foundation/moonbeam |
+|   `discord`   |                                               The project's Discord                                                |       https://discord.com/invite/PfpUATX        |
+
+The format of the property name/value pairs should follow the JSON standard, for example:
+
+```json
+--8<-- 'code/learn/dapps-list/dapp-directory/urls.json'
+```
+
+#### Logos {: #logos }
+
+The `logos` property of the main project data file is a map of image sizes (i.e., `small`, `large`, `full`) to corresponding image JSON objects. The image JSON object contains the display properties for the given image.
+
+The following table lists the properties of the image JSON object:
+
+|  Property  |  Type  |                               Description                                |
+|:----------:|:------:|:------------------------------------------------------------------------:|
+| `fileName` | String | The name of the image file (unqualified) stored in the `logos` directory |
+|  `width`   |  int   |                  The width of the logo image in pixels                   |
+|  `height`  |  int   |                  The height of the logo image in pixels                  |
+| `mimeType` | String |         The standard MIME type of the file. E.g., `"image/jpeg"`         |
```

Comment:
is there an URL that we can use to showcase what is a "MIME type?  for example [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)